src/main/java/Duke.java

```diff
@@ -1,10 +1,47 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String command;
+        String[] tasks = new String[100];
+        int taskNumber = 0;
+        boolean[] isDone = new boolean[100];
+
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+
+        loop: do {
```

Comment:
loop label is commonly used in C.. maybe you could exit at the case: "bye" instead of having a switch with a do-while loop? 💭